### PR TITLE
Add migrations to fix reimported blocked contacts

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by nicpottier** [comment]
I think we should do this at the same time that we fix blocked contacts being added to groups, otherwise we'll just get more inconsistent results.

**Comment by norkans7** [comment]
I think this is ready now

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1454,6 +1454,9 @@ def update_contacts(self, contacts, add):
 
             # if we are adding the contact to the group, and this contact is not in this group
             if add:
+                if contact.is_blocked:
+                    raise ValueError("Can't add or remove groups on blocked contact")
```
Comment (nicpottier) [inline temba/contacts/models.py]: So won't this blow up imports now?

Comment (norkans7) [inline temba/contacts/models.py]: actually imports do not use this method and at this point the contact is already unblocked

Comment (nicpottier) [inline temba/contacts/models.py]: Ah right, ok.

**Pull request merged**